qt/scientific_interfaces/Indirect/IndirectFittingModel.cpp

```diff
@@ -249,29 +249,18 @@ void cleanTemporaries(const std::string &base,
                       const std::unique_ptr<IndirectFitData> &fitData) {
   removeFromADSIfExists(base);
 
-  const auto clean = [&](std::size_t index, std::size_t /*unused*/) {
-    cleanTemporaries(base + "_" + std::to_string(index));
+  const auto clean = [&](IDAWorkspaceIndex index,
+                         IDAWorkspaceIndex /*unused*/) {
```

Comment:
If this param is un-used in the lambda can't it be removed?

Reply:
This lambda is passed in to another function which expects it to have inputs to call